languages.json

```diff
@@ -131,8 +131,24 @@
       "extensions": ["bash"]
     },
     "Batch": {
-      "line_comment": ["REM", "::"],
-      "extensions": ["bat", "btm", "cmd"]
+      "line_comment": [
```

Comment:
Second thought: this looks disgusting.

That said, I have no idea how to simplify it without significantly rewriting comment detection (as `tokei` performs binary comparison, so there is no notion of character casing), but doing that for one language only strikes me as wasteful and pointless.